### DISCUSSION TITLE
New dependency added to Android Calling SDK package.

### DISCUSSION THREAD

**Discussion by raosanat** [Laerdal/Xamarin.AzureCommunicationCalling#55] [General]
Hi,
 In an effort to support both Azure Communication Service Chat and Calling SDK's, we have added a new dependency to the Calling SDK package, [trouter](https://repo1.maven.org/maven2/com/microsoft/trouter-client-android/0.1.2/). Please update your bindings to support this new dependency.
 Pom file for the beta version of Azure Communication Calling SDK , [2.11.0-beta.1](https://repo1.maven.org/maven2/com/azure/android/azure-communication-calling/2.11.0-beta.1/azure-communication-calling-2.11.0-beta.1.pom).

**Comment by tompi** [comment]
Thank you for the heads up, I assume this applies to the ios acs client lib as well?

The binding lib needs some work, and I dont really have enough time to spend on it, but I will remember this next time I update.
